sickbeard/providers/torrentproject.py

```diff
@@ -108,13 +116,13 @@ def __init__(self, provider_obj):
 
         tvcache.TVCache.__init__(self, provider_obj)
 
-        # set this 0 to suppress log line, since we aren't updating it anyways
-        self.minTime = 0
+        # Use random number to avoid DDOSing the server
+        randomtime = random.randrange(10, 25)
+        self.minTime = randomtime
 
     def _getRSSData(self):
-        # no rss for torrentproject afaik,& can't search with empty string
-        # newest results are always > 1 day since added anyways
-        # search_strings = {'RSS': ['']}
-        return {'entries': {}}
+        
+        search_params = {'RSS': ['']}
```

Comment:
You can put num=150 in both RSS and non RSS, no need to selectively build the url and complicate the code.